frontend/src/components/dashboard/CodeComparison.tsx

```diff
@@ -83,6 +83,12 @@ const ModalFooter = styled.div`
   gap: 1rem;
 `;
 
+const CodeViewPlaceholder = styled.div`
+  height: 100vh;
```

Comment:
since this is supposed to be the same height as `CodeViewWrapper`, should we import the style from that other component?